packages/rpc-graphql/README.md

```diff
@@ -3,61 +3,1160 @@
+
+expect(result).toMatchObject({
+    data: {
+        account: {
+            address: 'AyGCwnwxQMCqaU4ixReHt8h5W4dwmxU7eM3BEQBdWVca',
+            data: {
+                parsed: {
+                    info: {
+                        mintAuthority: {
+                            address: expect.any(String),
+                            lamports: expect.any(BigInt),
+                        }
+                    }
+                }
+            },
+        },
+    },
+});
+```
+
+## Querying Program Accounts
+
+A very common way to query Solana accounts from an RPC is to request all of
+the accounts owned by a particular program.
+
+With RPC-GraphQL, querying program-owned accounts is a list-based extension of
+the account query defined previously. This means program accounts queries will
+return a list of objects implementing the `Account` interface.
+
+Setting up the query is very similar to the `account` query.
+
+```typescript
+const source = `
+    query myQuery($programAddress: String!) {
+        programAccounts(programAddress: $address) {
+            executable
+            lamports
+            rentEpoch
+        }
+    }
+`;
+
+const variableValues = {
+    programAddress: 'AmtpVzo6H6qQCP9dH9wfu5hfa8kKaAFpTJ4aamPYR6V6',
+};
+
+const result = await rpcGraphQL.query(source, variableValues);
+
+expect(result).toMatchObject({
+    data: {
+        // Array of `Account` objects
+        programAccounts: expect.arrayContaining([{
+            executable: expect.any(Boolean),
+            lamports: expect.any(BigInt),
+            rentEpoch: expect.any(BigInt),
+        }])
+    },
+});
+```
+
+### Specific Program Account Types
+
+Although specific parsed account types are directly tied to the program which
+owns them, it's still possible to handle various specific account types within
+the same program accounts response.
+
+```typescript
+const source = `
+    query myQuery($programAddress: String!) {
+        programAccounts(programAddress: $address) {
+            ... on MintAccount {
+                data {
+                    parsed {
+                        info {
+                            decimals
+                            isInitialized
+                            mintAuthority
+                            supply
+                        }
+                    }
+                }
+            }
+            ... on TokenAccount {
+                data {
+                    parsed {
+                        info {
+                            isNative
+                            mint
+                            owner
+                            state
+                        }
+                    }
+                }
+            }
+        }
+    }
+`;
+
+const variableValues = {
+    programAddress: 'AmtpVzo6H6qQCP9dH9wfu5hfa8kKaAFpTJ4aamPYR6V6',
+};
+
+const result = await rpcGraphQL.query(source, variableValues);
+
+const { mints, tokenAccounts } = log2.data.programAccounts.reduce(
```

Comment:
```suggestion
const { mints, tokenAccounts } = result.data.programAccounts.reduce(
```

Reply:
Lol! Some of these you caught me _not_ writing it in TS first, and others (like this) you caught me doing it and forgetting to change all the names.

Reply:
Thank you for this.